lib/api/bgppeer.go

```diff
@@ -37,6 +37,10 @@ func (t BGPPeer) GetResourceMetadata() unversioned.ResourceMetadata {
 	return t.Metadata
 }
 
+func (t BGPPeer) GetResourceID() string {
```

Comment:
I think you're better off having this method return a user friendly name that identifies this resource.  That way it could be returned to the user as well.  e.g. "BGP Peer (PeerIP=xxx, Scope=Global)"